src/main/java/namor/template/calculator/RPNcalculator.java

```diff
@@ -0,0 +1,71 @@
+package namor.template.calculator;
+
+import namor.template.stack.ArrayStack;
+
+public class RPNcalculator {
+    private ArrayStack numbers = new ArrayStack();
+    Sign sign;
+
+    public int calculate(String inputString){
+        for(String splitData : inputString.split(" ")){
+                A(splitData);
+        }
+        return numbers.pop();
+    }
+
+    private void A(String splitData) {
```

Comment:
I was talking a lot of times about naming, hope I see such a crap as 'A' for a last time ....